documentation/release-notes/19_3.md

```diff
@@ -0,0 +1,93 @@
+# run_statistics 
+
+---
+
+###Total number of reduction jobs for each instrument
+
+* GEM: 5688
+* HRPD: 986
+* MAPS: 1176
+* MARI: 545
+* MUSR: 1342
+* OSIRIS: 1185
+* POLARIS: 2810
+* POLREF: 650
+* WISH: 748
+
+*Total*
+15130
+
+---
+
+###Total number of unique runs for each instrument
+* ENGINX: 5688
+* GEM: 981
+* MAPS: 1074
+* MARI: 326
+* MUSR: 1340
+* OSIRIS: 1183
+* POLARIS,: 2798
+* POLREF: 646
+* WISH: 681
+
+*Total*
+15017
+
+---
+
+###Total number of Failed runs for each instrument
+
+* ENGINX: 40
```

Comment:
For the sake of being verbose, can we have 0 entries for the other instruments, e.g. GEM: 0